pkg/LinuxGeneralModule/lgm.go

```diff
@@ -477,6 +483,8 @@ func setUpVrf(vrf *infradb.Vrf) (string, bool) {
 			break
 		}
 		log.Printf("LGM: Routing Table %d is busy\n", routingTable)
+		addKey++
+		Name = fmt.Sprintf("%s%d", Name, addKey)
```

Comment:
1. style
2. If we use string concatenation to create names, i'd prefer it to be extracted into a separate function with clear name, what kind of id we cretate here

Reply:
if we use concatenation to form names, I'd prefer to get it extracted into a function which would have explanatory name, what kind of name we are constructing here

Reply:
removed the concatenation code. if the table id is busy we return error.